packages/mdc-button/_mixins.scss

```diff
@@ -66,9 +66,14 @@
   border-radius: $corner-radius;
 }
 
-@mixin mdc-button-stroke-width($stroke-width) {
+@mixin mdc-button-horizontal-padding($padding) {
+  padding-right: $padding;
```

Comment:
strictly speaking we should be adding /* @noflip */ on these. At the same time, I understand the argument about them being the same. Lets hope that never changes.

Reply:
Hm, yeah, I didn't think about it here since I know we handle `@noflip` within our RTL stuff now, and these styles are equal and not RTL-dependent. If these ever did change to being asymmetrical, we'd absolutely want to add `@noflip` (probably automatically via use of `mdc-rtl` mixins).

I think if we wanted to take that strict of a stance about `@noflip` we'd have several instances across the repo where left and right are equal that we're not flagging right now.